custom-routes-vis/src/main/webapp/script.js

```diff
@@ -320,7 +320,7 @@ function createRouteFromCoordinates(
 
 /** 
  * Highlights the selected route and displays its formatted duration and 
- * distance in miles.
+ * distance in miles. Displays a copy-route-token button if applicable.
```

Comment:
You want to say what the user will see.
```suggestion
 * distance in miles. Displays a "Copy Route Token" button if applicable.
```

Reply:
ok